testing/tests/DevExpress.ui.widgets/scrollView.tests.js

```diff
@@ -2133,166 +2125,6 @@ QUnit.test("scroll fires with correctly arguments", function(assert) {
-
-        var currentTopValue = 0;
-        renderer.fn.scrollTop = function(value) {
-            if(arguments.length) {
-                currentTopValue = value;
-            } else {
-                return currentTopValue;
-            }
-        };
-
-        this.originalJQueryScrollLeft = renderer.fn.scrollLeft;
-
-        var currentLeftValue = 0;
-        renderer.fn.scrollLeft = function(value) {
-            if(arguments.length) {
-                currentLeftValue = value;
-            } else {
-                return currentLeftValue;
-            }
-        };
-
-        moduleConfig.beforeEach.call(this);
-        this._originalPlatform = devices.real().platform;
-        devices.real({ platform: "win" });
-        $("#qunit-fixture").addClass("dx-theme-win8");
-    },
-    afterEach: function() {
-        renderer.fn.scrollTop = this.originalJQueryScrollTop;
-        renderer.fn.scrollLeft = this.originalJQueryScrollLeft;
-        moduleConfig.afterEach.call(this);
-        devices.real({ platform: this._originalPlatform });
-        $("#qunit-fixture").removeClass("dx-theme-win8");
-    }
-});
-
-QUnit.test("markup", function(assert) {
-    var $scrollView = $("#scrollView").dxScrollView({
-        useNative: true,
-        refreshStrategy: "slideDown",
-        onPullDown: noop
-    });
-
-    var $pullDown = $scrollView.find("." + SCROLLVIEW_PULLDOWN_CLASS),
-        $reachBottom = $scrollView.find("." + SCROLLVIEW_BOTTOM_POCKET_CLASS),
-        $content = $scrollView.find("." + SCROLLABLE_CONTENT_CLASS),
-        $topPocket = $scrollView.find("." + SCROLLVIEW_TOP_POCKET_CLASS);
-
-    assert.equal($pullDown.length, 0, "pull down container");
-    assert.equal($reachBottom.length, 1, "reach bottom container");
-    assert.equal($content.scrollTop(), $topPocket.height(), "content start on correctly position");
-});
-
-QUnit.test("onReachBottom", function(assert) {
-    assert.expect(1);
-
-    var scrollView = $("#scrollView").dxScrollView({
-        useNative: true,
-        refreshStrategy: "slideDown",
-        onReachBottom: function() {
-            assert.ok(true, "onReachBottom action was fired");
-        }
-    }).dxScrollView("instance");
-
-    var $container = scrollView.$element().find("." + SCROLLABLE_CONTAINER_CLASS);
-
-    $container.scrollTop($container.prop("scrollHeight") - $container.prop("clientHeight"));
-    $($container).trigger("scroll");
-});
-
-QUnit.test("reach bottom element toggle", function(assert) {
-    var $scrollView = $("#scrollView").dxScrollView({
-        useNative: true,
-        refreshStrategy: "slideDown"
-    });
-
-    var $reachBottom = $scrollView.find("." + SCROLLVIEW_BOTTOM_POCKET_CLASS);
-
-    assert.ok($reachBottom.is(":hidden"), "reach bottom is hidden");
-
-    $scrollView.dxScrollView("option", "onReachBottom", noop);
-
-    assert.ok($reachBottom.is(":visible"), "reach bottom is hidden");
-});
-
-QUnit.test("release fires update", function(assert) {
```

Comment:
This test doesn't look related to the old devices, why are we going to remove it?